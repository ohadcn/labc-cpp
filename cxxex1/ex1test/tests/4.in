264,-5792
2545,-7815
-8478,-5301
-679,-3986
6891,3152
-5731,2121
-4950,-5910
-1091,-8146
1613,9201
-9791,5926
-3832,-1066
5755,-5560
2478,-8666
-842,-8864
-6896,-5194
8191,7089
2735,736
9274,4257
5435,2316
-9730,6046
-4533,-1740
-1833,518
2351,-9204
2372,-2316
9997,6302
-6391,-3835
-4765,-6915
4326,-8567
-5581,-6517
-7430,1244
2010,-9240
-1668,-5256
5217,-2394
9001,4372
9922,-7008
4139,-891
1252,-3974
9627,-2677
-3178,1998
5007,539
2021,-7663
-9576,7256
-857,8471
-7590,7283
-8047,-1299
-1474,7684
3182,6858
-7573,2120
-1816,-4851
-3509,-1895
-1859,-9371
7215,-6887
376,-9439
4158,-9082
6281,2885
1458,-1699
5222,5603
-723,-1915
7795,1687
-4633,-6532
-9613,3893
-8848,7291
4472,-2700
-590,-7345
2449,5901
4482,-5690
-9749,1696
1145,627
5978,-4698
1546,2258
8187,-3276
-9442,7130
-3953,-164
-4786,-6159
-8478,582
1031,-4369
-1805,2183
-7079,-7334
9483,2331
-4678,5653
1952,9804
-6317,5924
5220,4828
272,1197
-9871,-4462
-2824,-7963
2262,-2266
9167,-7971
7570,-1899
-408,-7187
-7597,-9378
2165,599
2805,8807
6986,6008
1137,-3972
5381,-3191
-4169,9065
-7267,-5229
3892,6726
9690,7743
