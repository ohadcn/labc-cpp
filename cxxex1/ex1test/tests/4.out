result
-9871,-4462
-9791,5926
-9576,7256
-7597,-9378
-4169,9065
376,-9439
1952,9804
4158,-9082
9167,-7971
9690,7743
9922,-7008
9997,6302
